integration-tests/tests/mainnet.rs

```diff
@@ -81,11 +81,12 @@ async fn test_mock_prove_tx() {
     }
 
     let mut block = block_convert::<Fr>(&builder.block, &builder.code_db).unwrap();
+    #[cfg(feature = "scroll")]
     witness::block_mocking_apply_mpt(&mut block);
```

Comment:
why don't we just remove this since we are running the ethereum mainnet integration test?

Reply:
Seems the `Cargo.toml` of integration-tests has [a scroll feature](https://github.com/scroll-tech/zkevm-circuits/blob/develop/integration-tests/Cargo.toml#L37).

And there are e.g. commands uses `--features scroll` in the [README.md](https://github.com/scroll-tech/zkevm-circuits/tree/develop/integration-tests/tests#mainnet) of integration-tests as:
<img width="600" alt="Screen Shot 2023-11-22 at 8 56 48 AM" src="https://github.com/scroll-tech/zkevm-circuits/assets/31645658/13f8d1b3-11cb-491f-929a-4a623408aa71">

cc @noel2004

Reply:
After discussing with @kunxian-xia and @noel2004 , we suppose to test with l2geth RPC later (when EIP 1559 is enabled), the MPT could also be tested with `scroll` feature.